package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "wallet-vc-libs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Are we going to have this `index.js` entrypoint?

Reply:
No not at this level. An entry point is not needed here. Removed.